akka-persistence/src/main/scala/akka/persistence/journal/ReplayFilter.scala

```diff
@@ -30,6 +30,13 @@ private[akka] object ReplayFilter {
     Props(new ReplayFilter(persistentActor, mode, windowSize, maxOldWriters, debugEnabled))
   }
 
+  // for binary compatibility
+  def props(
+    persistentActor: ActorRef,
+    mode: Mode,
+    windowSize: Int,
+    maxOldWriters: Int): Props = props(persistentActor, mode, windowSize, maxOldWriters, debugEnabled = false)
```

Comment:
Yes, of course. All I’m saying is that the cost of breaking compatibility without thinking twice is also not zero—for internal api the cost might be small, which means that a small amount of burden might be justified.